WORKSPACE

```diff
@@ -174,14 +174,14 @@ distdir_tar(
             "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/archive/f83cb8dd6f5658bc574ccd873e25197055265d1c.tar.gz",
             "https://github.com/bazelbuild/bazel-skylib/archive/f83cb8dd6f5658bc574ccd873e25197055265d1c.tar.gz",
         ],
-        "java_tools_javac11_linux-v5.1.zip": [
-            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_linux-v5.1.zip",
+        "java_tools_javac11_linux-v6.0.zip": [
+            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_linux-v6.0.zip",
         ],
-        "java_tools_javac11_windows-v5.1.zip": [
-            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_windows-v5.1.zip",
+        "java_tools_javac11_windows-v6.0.zip": [
+            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_windows-v6.0.zip",
         ],
-        "java_tools_javac11_darwin-v5.1.zip": [
-            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_darwin-v5.1.zip",
+        "java_tools_javac11_darwin-v6.0.zip": [
+            "https://mirror.bazel.build/bazel_java_tools/releases/javac11/v5.1/java_tools_javac11_darwin-v6.0.zip",
```

Comment:
You missed to replace the directory name, it is stilled v5.1 and should be v.6.0